### DISCUSSION TITLE
Add color picker for colors inside tailwind.config.js

### DISCUSSION THREAD

**Discussion by naptestdev** [tailwindlabs/tailwindcss#6009] [Ideas]
Add color picker like this

![image](https://user-images.githubusercontent.com/88878684/140730267-38909069-1b0c-4f2b-a95b-f27f133b06fd.png)

Inside tailwind.config.js
![image](https://user-images.githubusercontent.com/88878684/140730333-73b681e6-8a5b-43c1-a226-daf28eeb27bb.png)

**Comment by NovusOpus** [comment]
Updated: Tailwind CSS IntelliSense Already support color picker by hovering the Color preview square.

**Reply by Jugibur** [reply]
Unfortunately, I still don't see a color preview / color square in the configuration file, even when I enter a #. 
Some ideas how to fix that?

**Reply by CheersLoveDani** [reply]
The VSC extension [Color Picker Universal](https://marketplace.visualstudio.com/items?itemName=JeronimoEkerdt.color-picker-universal) solves this issue and allows color picking in strings.
